packages/grant-explorer/src/features/common/Navbar.tsx

```diff
@@ -16,6 +19,21 @@ export default function Navbar(props: NavbarProps) {
   const showWalletInteraction = props.showWalletInteraction ?? true;
   const currentOrigin = window.location.origin;
 
+  const { roundId } = useParams();
+
+  useEffect(() => {
+    const storageEventHandler = (event: StorageEvent) =>
+      // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
you can surround the body of this function with an `if(roundId) { }` instead of doing a null assertion